modules/installation-configuration-parameters.adoc

```diff
@@ -57,10 +57,10 @@ ifeval::["{context}" == "installing-aws-customizations"]
 :aws:
 endif::[]
 ifeval::["{context}" == "installing-aws-government-region"]
-:aws:
+:aws-non-arm:
```

Comment:
```suggestion
:aws:
:aws-govcloud:
```

Reply:
@yselkowitz I was thinking that once something is supported (whether its UPI, or government) the `ifdevel` can just be switched from `aws-non-arm` to `aws`, or do you think the more specific names are more clear?